pkgs/applications/window-managers/sway/contrib.nix

```diff
@@ -58,21 +66,20 @@ grimshot = stdenv.mkDerivation rec {
 
   meta = with lib; {
     description = "A helper for screenshots within sway";
-    homepage = "https://github.com/swaywm/sway/tree/master/contrib";
+    homepage = "https://github.com/OctopusET/sway-contrib";
```

Comment:
Could theoretically be inherited from `src.meta` but it seems like this might not be preferred anymore.